src/functionaltests/java/com/ericsson/ei/subscriptions/bulk/SubscriptionBulkSteps.java

```diff
@@ -61,7 +61,7 @@ public void make_a_GET_request_with_list_of_subscriptions_names_to_the_subscript
             String subscriptionsNamesList, String endpoint) throws Throwable {
 
         HttpRequest getRequest = new HttpRequest(HttpMethod.GET);
-        response = getRequest.setHost(hostName).setPort(port).setEndpoint(endpoint + "/" + subscriptionsNamesList)
+        response = getRequest.setHost(hostName).setPort(port).setEndpoint(endpoint + "?subscriptionName=" + subscriptionsNamesList)
```

Comment:
Does the search parameter name have to be "subscriptionName"? For me it feels both a bit redundant since the endpoint path already contains "subscriptions" in it, and I think can be easy to misspell ... how abut shortening it to just "?name=" ?

Reply:
I used name to start with, but then I realized that the field in the subscription object is called subscriptionName so I thought it would be an exact match but I can change it if you think it's better.

Reply:
Alright, fair enough. I also remembered we will probably want the same search parameter for the new endpoint failed-notifications anyway to clarify that we mean subscriptionName and not something else. (although current table in issue #291 says names)

Reply:
So after talking more about this the parameter is now called subscriptionNames in plural.

Reply:
I updated the endpoint table.